services/slam/builtin/orbslam_int_test.go

```diff
@@ -153,13 +159,27 @@ func integrationTestHelper(t *testing.T, mode slam.Mode) {
 				break
 			}
 			test.That(t, strings.Contains(line, "Fail to track local map!"), test.ShouldBeFalse)
+			if time.Since(start_time_sent_image) > time.Duration(dataInsertionMaxTimeoutMin)*time.Minute {
+				orbslam_hangs = true
+				t.Log("orbslam hangs")
+				break
+			}
+		}
+		if orbslam_hangs {
```

Comment:
[minor] move to beginning of the for loop so that an extra check isn't performed after timeout. Otherwise this could result in the log 'orbslam hangs' appearing and then Passed image to SLAM

Reply:
I don't understand. This line is executed right after the timeout happens.

Reply:
great point!